server/user.go

```diff
@@ -43,15 +43,30 @@ func (c *Connection) JiraAccountID() types.ID {
 }
 
 type ConnectionSettings struct {
-	Notifications bool `json:"notifications"`
+	Notifications bool  `json:"notifications"`
+	Watching      *bool `json:"watching"`
 }
 
 func (s *ConnectionSettings) String() string {
-	notifications := "off"
-	if s != nil && s.Notifications {
-		notifications = "on"
+	notifications, watching := "off", "off"
+	if s != nil {
+		if s.Notifications {
+			notifications = "on"
+		}
+		if s.Watching == nil || *s.Watching {
+			watching = "on"
+		}
+	}
+	return fmt.Sprintf("\tNotifications: %s\n\tWatching: %s", notifications, watching)
```

Comment:
```suggestion
	return fmt.Sprintf("- Notifications: %s\n- Watching: %s", notifications, watching)
```

Which will produce:

- Notifications: on
- Watching: on